docs/source/scancode-toolkit/getting-started/whats-new.rst

```diff
@@ -38,8 +38,8 @@ Brief Summary Of Changes
 Documentation Support
 ^^^^^^^^^^^^^^^^^^^^^
 
-``pip install``
-^^^^^^^^^^^^^^^
+``pip install`` Support
+^^^^^^^^^^^^^^^^^^^^^^^
 
 Python 3 Support
 ^^^^^^^^^^^^^^^^
```

Comment:
I'd also need suggestions on this file, i.e. `scancode-toolkit/getting-started/whats-new.rst`. Basically on this `Brief Summary Of Changes` section, should it have more points? Does this page make sense? The [ToDo] comment in this [file](https://github.com/nexB/aboutcode/blob/master/docs/source/scancode-toolkit/getting-started/whats-new.rst) explains what I'm proposing.

Reply:
We do have a CHANGELOG file here: https://github.com/nexB/scancode-toolkit/blob/develop/CHANGELOG.rst

Maybe all we need to do is link to it in `whats-new.rst`

Reply:
@majurg Yes, but my thoughts were that it contains a lot of information about bugs and every other small change too. I did obviously link to the changelog file, you can check [here](https://aboutcode-test.readthedocs.io/en/update-docs/scancode-toolkit/getting-started/whats-new.html#main-new-features). This page is a more broad overview to the users, explaining why they should this specific version.

Reply:
On a side note, why doesn't the changelog include version 3.1.1?

Reply:
The changelog will eventually be updated I believe.